docs/configuration.html

```diff
@@ -286,7 +286,12 @@ <h4 class="anchor-heading"><a id="mirrormakerheartbeatconfigs" class="anchor-lin
   Below is the configuration of MirrorMaker 2 heartbeat connector for checking connectivity between connectors and clusters.
   <!--#include virtual="generated/mirror_heartbeat_config.html" -->
 
-  <h3 class="anchor-heading"><a id="systemproperties" class="anchor-link"></a><a href="#systemproperties">3.9 System Properties</a></h3>
+  <h3 class="anchor-heading"><a id="remotestorageconfigs" class="anchor-link"></a><a href="#remotestorageconfigs">3.9 Tier Storage Configs</a></h3>
+  Below is the configuration of Tier Storage.
```

Comment:
```suggestion
  <h3 class="anchor-heading"><a id="remotestorageconfigs" class="anchor-link"></a><a href="#remotestorageconfigs">3.9 Tiered Storage Configs</a></h3>
  Below are the configuration properties for Tiered Storage.
```

Reply:
I noticed that some of the config docs seem to refer to tier storage. Perhaps I should update those?